### PR TITLE
Add signup-form feature

### PR DESCRIPTION
**Pull request by OoOHAMiDOoO**
Add signup-form feature:
Implement signup form to project.

### CODE REVIEW

**Review by HAW1D — CHANGES_REQUESTED** [review]
Please implement feature, there is not code.

File: signup.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Signup form
```
Comment (OoOHAMiDOoO) [inline signup.txt:1]: That's ok

**Review by HAW1D — APPROVED** [review]
That's Great

**Review by OoOHAMiDOoO — COMMENTED** [review]
Implement

**Pull request merged**